docs/trex_create.md

```diff
@@ -172,19 +172,18 @@ Your web application must include an HTML page. This page should link to the Ext
 
 After you have created the manifest file (`.trex`) and have hosted your web app you can test it in Tableau. It's a good idea to do this even if your application isn't completed.
 
-
 1. Start up your web page or application (or make sure it is running).
 
-2. Start Tableau and open a workbook with a dashboard or create a new dashboard.
-3. In the dashboard, under **Objects**, select **Extension** and drag it on to the dashboard. In the **Choose an Extension** dialog box, click **My Extensions** and browse to directory where you have your manifest file.
+2. Start Tableau and open a workbook with a dashboard, or create a new dashboard with at least one worksheet.
```

Comment:
@johnDance @bcantoni  same story here. We probably don't need to say "with at least one worksheet"  When you open a new workbook, it automatically creates a worksheet. In this example, where we just get the name of the dashboard, it doesn't matter if it's blank.

Reply:
Yes, having it in there makes it sound like a requirement. While most extensions need a worksheet, it isn't a requirement.

By the way, I only tagged one of these statement. There was at least 1 above here, or maybe two.

Reply:
@johnDance  Correct. There were two places where I added the "with at least one worksheet."  I've removed that from both. In this case (trex_create.md), I just left it the way it was "open a workbook  with a dashboard, or create an new dashboard.."